terminologies/technique/glutamateReceptor2Staining.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/glutamateReceptor2Staining",

```

Comment:
is this not one of the ones we wanted to reduce to primary and secondary antibody staining? I've may have missed some above as well...

Reply:
Yes. Prim/Sec antibody staining. Looked it up in KG. Changed to 
technique: "primary antibody staining technique" + "secondary antibody staining technique"
studyTarget: "AMPA receptor expressing neurons"
input: "anti-glutamate receptor 2 antibody"